master/reference/cni-plugin/configuration.md

```diff
@@ -76,6 +76,12 @@ When using Calico IPAM, the following flags determine what IP addresses should b
 
 A specific IP address can be chosen by using [`CNI_ARGS`](https://github.com/appc/cni/blob/master/SPEC.md#parameters) and setting `IP` to the desired value.
 
+The list of possible IPv4 and IPv6 pools can also be specified via the following properties:
+* `ipv4_pools`: An array of CIDR strings (e.g. `{ "10.0.0.1/24", "20.0.0.1/16"}`
```

Comment:
Take a look at this comment - I think this is the desired format: https://github.com/projectcalico/cni-plugin/issues/213#issuecomment-272053112